ios/Runner/Info.plist

```diff
@@ -10,18 +10,22 @@
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
+  <key>NSPhotoLibraryUsageDescription</key>
+  <string>We need to access your photo library so you can choose your own pictures when creating your Memos</string>
 	<key>CFBundleName</key>
-	<string>memo</string>
+	<string>Memo</string>
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(FLUTTER_BUILD_NAME)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>2</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+    <key>ITSAppUsesNonExemptEncryption</key>
+    <false/>
```

Comment:
What is the reason for this?

Reply:
Every time we submit an iOS app to review, Apple ask if the app is using any kind of encryption beyond those that are exempt from the export compliance. Since we don't use, and not even plan to use, I added this property to Info.plist indicating that we don't use encryption to avoid Apple asking for encryption every time we submit a new build

See for more info: https://developer.apple.com/documentation/bundleresources/information_property_list/itsappusesnonexemptencryption